0-bootstrap/README-Jenkins.md

```diff
@@ -284,6 +284,12 @@ Here you will configure a VPN Network tunnel to enable connectivity between the
 
 1. You can now move to the instructions in the step [1-org](../1-org/README.md).
 
+### Optional Variables
```

Comment:
This readme is getting pretty big. Thoughts on a Table of contents with anchors?

Reply:
+1